scripts/p2p-test-tool.py

```diff
@@ -0,0 +1,403 @@
+
+    for network in client.networks.list():
+        if args_network == network.name:
+            print(f"removing {network.name}")
+            network.remove()
+    # client.volumes.prune() # removes unused volumes
+    return 0
+
+
+def create_bootstrap_node():
+    """Create bootstrap node."""
+    bootstrap_node = {}
+    bootstrap_node['name'] = f"bootstrap.{args.network}"
+    bootstrap_node['volume'] = client.volumes.create()
+    print(f"creating {bootstrap_node['name']}")
+    container = client.containers.run(args.image, \
+        name=bootstrap_node['name'], \
+        detach=True, \
+        cpuset_cpus=args.cpuset_cpus, \
+        mem_limit=args.memory, \
+        network=args.network, \
+        volumes={bootstrap_node['volume'].name: {'bind': args.rnode_directory, 'mode': 'rw'}}, \
+        command=args.bootstrap_command, \
+        hostname=bootstrap_node['name'])
+
+    # Add additional packages.
+    container.exec_run(cmd='apk update')
+    container.exec_run(cmd='apk add curl')
+    container.exec_run(cmd='apk add nmap')
+    container.exec_run(cmd='apk add python3')
+    container.exec_run(cmd='pip3 install requests')
+    return 0
+
+
+def create_peer_nodes():
+    """Create peer nodes."""
+    for i in range(args.peers_amount):
+        peer_node = {}
+        peer_node[i] = {}
+        peer_node[i]['name'] = f"peer{i}.{args.network}"
+        peer_node[i]['volume'] = client.volumes.create()
+        print(f"creating {peer_node[i]['name']}")
+        container = client.containers.run(args.image, \
+            name=peer_node[i]['name'], \
+            detach=True, \
+            cpuset_cpus=args.cpuset_cpus, \
+            mem_limit=args.memory, \
+            network=args.network, \
+            volumes=[f"{peer_node[i]['volume'].name}:{args.rnode_directory}"], \
+            command=args.peer_command, \
+            hostname=peer_node[i]['name'])
+
+        # Add additional packages.
+        container.exec_run(cmd='apk update')
+        container.exec_run(cmd='apk add curl')
+        container.exec_run(cmd='apk add nmap')
+        container.exec_run(cmd='apk add python3')
+        container.exec_run(cmd='pip3 install requests')
+    return 0
+      
+
+def test_network_sockets(container):
+    print(f"Test metrics api socket for {container.name}")
+    try:
+        cmd = f"nmap -sS -n -p T:9095 -oG - {container.name}"
+        r = container.exec_run(cmd=cmd).output.decode("utf-8")
+        if "9095/open/tcp" in r:
+            return 0 
+        else:
+            return 1 
+    except Exception as e:
+        print(e)
+        return 1 
+
+
+def test_repl_load(container):
+    """Load REPL with commands."""
+    for repl_container in client.containers.list(all=True, filters={"name":f"repl\d.{args.network}"}):
+        print(f"removing {repl_container.name}")
+        repl_container.remove(force=True, v=True)
+    i = 0
+    try:
+        repl_node = {}
+        repl_node[i] = {}
```

Comment:
This is never anything but 0 right?

Reply:
Yes. I just put it in there if you wanted to do more later. You could loop through them. I can remove it out.